### PR TITLE
Disclaimer for alpha development

### PR DESCRIPTION
**Pull request by sfriedowitz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,13 @@ for compute scaling, or an
 with inference performed via an in-process HuggingFace model or an externally-hosted 
 [vLLM](https://github.com/vllm-project/vllm) server.
 
+### Warning
+
+LM Buddy is in the early stages of development.
+It is missing important features and documentation.
+You should expect breaking changes in the configuration structures and core interfaces
+until the first major release.
```
Comment (veekaybee) [inline README.md:20]: should we add in something about following semver/releases here?

Comment (sfriedowitz) [reply README.md:20]: Happy to change this wording to be agnostic to our versioning scheme if you have a better way to word this?

Comment (veekaybee) [reply README.md:20]: Yeah basically something that lets people know just how early this is, aka what is a "major release" in relation to where we are now. Maybe there are some projects also in early stages whose wording we can take a look at.

**Comment by sfriedowitz** [comment]
@veekaybee I've changed the wording to be agnostic to our release schedule. Is this better?

**Comment by veekaybee** [comment]
Yep, works for me. We might also want to add the [info or warning syntax](https://github.com/orgs/community/discussions/16925)

**Review by binaryaaron — APPROVED** [review]

**Pull request merged**